Surface_mesh/include/CGAL/Surface_mesh/Surface_mesh.h

```diff
@@ -2502,49 +2504,171 @@ class Property_map
   inline std::istream& sm_skip_comments( std::istream& in) {
       char c;
       in >> c;
-      if (c == '#')
-        in.ignore((std::numeric_limits<std::streamsize>::max)(), '\n');
-      else
-        in.putback(c);
+      while(c == '#')
+      {
+       in.ignore((std::numeric_limits<std::streamsize>::max)(), '\n');
+       in >> c;
+      }
+      in.putback(c);
       return in;
   }
 /// @endcond
 
 
+  inline CGAL::Color get_color_from_line(std::istream &is)
```

Comment:
There is already a very similar function with the same name in `<CGAL/IO/File_scanner_OFF.h>`.